pycytominer/cyto_utils/collate_cmd.py

```diff
@@ -0,0 +1,92 @@
+if __name__ == "__main__":
+    import argparse
+    from pycytominer.cyto_utils.collate import collate
+
+    parser = argparse.ArgumentParser(description="Collate CSVs")
```

Comment:
If you decide to accept some of my suggestions about renaming variables, you'll likely have to update some of these arguments and helper descriptions